pages/windows/iscc.md

```diff
@@ -0,0 +1,16 @@
+# iscc
+
+> Compiler for Inno Setup installers.
+> It compiles an Inno Setup scripts into an Windows installer executable.
+
+- Compile an Inno Setup script:
+
+`iscc {{path/to/file.iss}}`
+
+- Quietly compile an Inno Setup installer:
+
+`iscc /Q {{path/to/file.iss}}`
+
+- Compile a signed Inno Setup installer:
+
+`iscc /S={{name}}={{command}} {{path/to/file.iss}}`
```

Comment:
What do `{{name}}` and `{{command}}` do here?

Reply:
It's a terrible syntax and can look like this:

`{{name}}` can be anything a user chooses, the name will then be used in an Inno Setup script.

**Example:**

```ini
SignTools=mySignTool
```

I haven't worked with signed installers before, so I can't really explain why one needs to define the name. Maybe so you can sign installer and uninstallers differently?

`{{command}}` is the command to sign the executable, e.g. `C:\Program Files\Microsoft Visual Studio 8\SDK\v2.0\Bin\signtool.exe sign /f CertPath\mycert.pfx /p myPassword $f"`

Put together in the command-line, it looks like this:

`iscc "/SmySignTool=C:\Program Files\Microsoft Visual Studio 8\SDK\v2.0\Bin\signtool.exe sign /f CertPath\mycert.pfx /p myPassword $f" myInstaller.iss`

Reply:
Right, I think I see. Maybe it's for multiple signatures or something? Anyway, thanks for clearing that up :smiley_cat: